### PR TITLE
Season Switch additions

### PR DESCRIPTION
**Pull request by F3PiX**
Solves open to do's for Season Phase Switch

### CODE REVIEW

File: app/models/season.rb [hunk 1]
```diff
@@ -47,7 +47,8 @@ def fake_coding_phase
   end
 
   def back_to_reality
-    update({
+    # OPTIMIZE change update to destroy and adjust UX for developers accordingly
+  update({
```
Comment (carpodaster) [inline app/models/season.rb]: Indentation

File: app/models/season.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ def started?
 
   def year; name end
 
-  # switch_phase, in dev env only: enables developers to easily switch
+  # switch_phase, not available in deployment: enables developers to easily switch
   # between time dependent settings in views
```
Comment (carpodaster) [inline app/models/season.rb]: Technically, the action on the model instance is available anywhere. The controller action isn't ;)

**Comment by carpodaster** [comment]
Hey Maud, feel like adding a unit test for the methods added on `Season`? :wink:  Would be great, but we can also merge without it.

**Comment by carpodaster** [comment]
Hey @F3PiX, I'm leaving this open as a reminder that we wanted to add some tests for the season switch. There's also an indentation issue pending.

**Comment by carpodaster** [comment]
Hello @F3PiX – would you still like to add the mentioned unit test for the low level swicher logic?

**Comment by F3PiX** [comment]
Hello @carpodaster. Let me have another shot at this.

**Comment by F3PiX** [comment]
@carpodaster I messed up the forked branch. To avoid bringing the mess into Upstream, I created a fresh PR #369. Can I close this?

**Comment by carpodaster** [comment]
@F3PiX Sure, close it at you leisure.

**Pull request closed without merging**